ktlint-reporter-plain/src/main/kotlin/com/pinterest/ktlint/reporter/plain/PlainReporterProvider.kt

```diff
@@ -13,9 +14,16 @@ class PlainReporterProvider : ReporterProvider {
             out,
             verbose = opt["verbose"]?.emptyOrTrue() ?: false,
             groupByFile = opt["group_by_file"]?.emptyOrTrue() ?: false,
-            color = opt["color"]?.emptyOrTrue() ?: false,
+            color = getColor(opt["color"]),
             pad = opt["pad"]?.emptyOrTrue() ?: false
         )
 
     private fun String.emptyOrTrue() = this == "" || this == "true"
+
+    private fun getColor(colorInput: String?): Color? {
+        return when (colorInput) {
+            "null" -> null
+            else -> Color.values().firstOrNull { it.name == colorInput } ?: Color.DARK_GRAY
```

Comment:
Not sure, maybe it's better not to fallback to gray color here, but behave like the value wasn't supplied? Or maybe even throw an exception that the argument is not matched.